frontend/i18n/locales/en.json

```
@@ -907,6 +907,8 @@
     "pricing": {
         "addons": {
             "button": {
+                "decrease_amount": "Decrease amount",
```

question: isn't it rather the `quantity` of the `product`? 
```suggestion
                "decrease_quantity": "Decrease quantity",
```

---

You're right, changed